initrd/directory.go

```diff
@@ -65,6 +65,12 @@ func (initrd *directory) Build(ctx context.Context) (string, error) {
 		}
 	}
 
+	if _, err := os.Stat(initrd.opts.output); err == nil {
+		if err = os.Remove(initrd.opts.output); err != nil {
+			return "", err
+		}
+	}
```

Comment:
This can be potentially dangerous operation since it is hidden away. Instead, simply error out if the output already exists instead of removing it:

```go
if _, err := os.Stat(initrd.opts.output); err == nil {
  return "", fmt.Errorf("output already exists")
}
```

Reply:
Done 👍🏽

Reply:
Also, It causes test failure, So should I update the test as well ?